tram-one.js

```diff
@@ -40,10 +40,21 @@ class Tram {
     return this
   }
 
-  addRoute(path, page) {
+  addRoute(path, page, subroutes) {
     assert.equal(typeof path, 'string', 'Tram-One: path should be a string')
     assert.equal(typeof page, 'function', 'Tram-One: page should be a function')
 
+    if (subroutes) {
+      subroutes.forEach(subroute => {
+        const newPath = path + subroute.path
+        const newPage = (store, actions, params, child) => {
```

Comment:
Would make this function name clear that it is *doing* something. Good rule of thumb is always have a verb with function names, especially when using `const`. This makes it easier to reason what its purpose is.

Reply:
Maybe `makeNewPage`